.github/workflows/static.yml

```
@@ -0,0 +1,29 @@
+name: Check collectstatic
+on: [push, pull_request]
```

Allows triggering a build from the UI:
```suggestion
on: [push, pull_request, workflow_dispatch]
```

---

Is this necessary? Its mainly just a check to ensure that a code change doesn't break our collect static step.

---

Nope, definitely not necessary.

I generally just add it to most workflows, because when I do want to trigger one, it can be easier to click a button than make a dummy commit or re-run an old one.

Feel free to skip if it's not useful here.